moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -918,6 +941,15 @@ bool ServoCalcs::checkValidCommand(const geometry_msgs::msg::TwistStamped& cmd)
     }
   }
 
+  // Check that the command frame is known
+  if (!current_state_->knowsFrameTransform(cmd.header.frame_id))
+  {
+    rclcpp::Clock& clock = *node_->get_clock();
+    RCLCPP_WARN_STREAM_THROTTLE(LOGGER, clock, ROS_LOG_THROTTLE_PERIOD,
```

Comment:
Nit: this too can be simplified by rewriting: `RCLCPP_WARN_STREAM_THROTTLE(LOGGER, *node_->get_clock(), ROS_LOG_THROTTLE_PERIOD, etc...`

Reply:
This actually does not work. I don't understand exactly how ROS 2 clocks and timing works, but this has weird results:

It will compile, but some timestamps (seen in RCLCPP logging macros) turn into... not quite `nullptr` ? It is very strange